docs/user-guide/ingest-data/for-observability/prometheus.md

```diff
@@ -156,6 +156,39 @@ WHERE greptime_timestamp > "2024-08-07 03:27:26.964000"
   AND job = "job1";
 ```
 
+## Special labels for ingestion options
+
+:::warning Experimental Feature
+This experimental feature may contain unexpected behavior, have its functionality change in the future.
+:::
+
+Originally, the whole data of a remote write request is ingested into the database under the same option(e.g, `db_name`).
+For advanced users and large data volume, it might be helpful to split the storage during ingestion to improve query efficiency and speed.
+
+Starting from `v0.15`, GreptimeDB is adding support for special labels.
+There labels(along with there values) will turn into ingest options during the parsing phase, allowing individual metric within a request to be more precisely controlled.
+
+The labels are not mutually exclusive, so they can be combined together to produce more versatile controlling.
+
+Note, if you're not encountering any problems using GreptimeDB with metrics, or you don't quite understand the point of these options, you probably won't need them anyway.
```

Comment:
if readers don't understand these options, it means that the documentation does not describe these options clearly.